pkgs/development/python-modules/kornia-rs/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, rustPlatform
+, cmake
+, nasm
+}:
+
+buildPythonPackage rec {
+  pname = "kornia-rs";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "kornia";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-7toCMaHzFAzm6gThVLBxKLgQVgFJatdJseDlfdeS8RE=";
+  };
+
+  format = "pyproject";
+
+  nativeBuildInputs = [
+    rustPlatform.maturinBuildHook
+    rustPlatform.cargoSetupHook
+    cmake # Only for dependencies.
+    nasm # Only for dependencies.
+  ];
+
+  cargoRoot = "py-kornia";
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = ./Cargo.lock;
+  };
+
+  prePatch = ''
+    # The path dependency doesn't vendor the dependencies correctly, so get kornia-rs from crates instead.
+    patch -p1 <<END_PATCH
+    --- a/py-kornia/Cargo.toml
+    +++ b/py-kornia/Cargo.toml
+    @@ -9,6 +9,6 @@ crate-type = ["cdylib"]
+
+     [dependencies]
+     anyhow = "1.0.80"
+    -kornia-rs = { path = ".." }
+    +kornia-rs = { version = "${version}" }
+     pyo3 = { version = "0.20", features = ["extension-module"] }
+     numpy = { version = "0.20.0" }
+
+    END_PATCH
+
+    cp ${./Cargo.lock} py-kornia/Cargo.lock
+  '';
+
+  maturinBuildFlags = [ "-m" "py-kornia/Cargo.toml" ];
+
+  dontUseCmakeConfigure = true; # We only want to use CMake to build some Rust dependencies.
+
+  meta = with lib; {
+    homepage = "https://github.com/kornia/kornia-rs/tree/main";
+    description = "Python bindings to Low-level Computer Vision library in Rust";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ bcdarwin ];
```

Comment:
This should be you unless @bcdarwin agreed to take over maintainership.

Reply:
It's a helper package for `kornia` though, which he maintains.

Reply:
Why is this still not you?

Reply:
See the comment above, I didn't get an answer yet.

Reply:
So why do you ask me to take a look while this isn't fixed yet?

Reply:
I asked @bcdarwin too, and I wanted to know if you had any other issues.